cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split1/FormRecognizerV3Suite.scala

```diff
@@ -0,0 +1,210 @@
+        "LastName,Locale,Region,Restrictions,Sex")
+
+    val result2 = modelsTest(bytesAnalyzeIDDocuments, bytesDF5, true)
+    resultAssert(result2, "USA WASHINGTON\nWA\n20 1234567XX1101\nDRIVER LICENSE\nFEDERAL LIMITS APPLY",
+      "Address,CountryRegion,DateOfBirth,DateOfExpiration,DocumentNumber,Endorsements,FirstName," +
+        "LastName,Locale,Region,Restrictions,Sex")
+  }
+
+  lazy val analyzeBusinessCards: AnalyzeDocumentV3 = analyzeDocumentV3
+    .setPrebuiltModelId("prebuilt-businessCard")
+    .setImageUrlCol("source")
+
+  lazy val bytesAnalyzeBusinessCards: AnalyzeDocumentV3 = analyzeDocumentV3
+    .setPrebuiltModelId("prebuilt-businessCard")
+    .setImageBytesCol("imageBytes")
+
+  test("Prebuilt-businessCard Basic Usage") {
+    val result1 = modelsTest(analyzeBusinessCards, imageDf3, false)
+    resultAssert(result1, "Dr. Avery Smith\nSenior Researcher\nCloud & Al Department\navery.smith@contoso.com\n" +
+      "https://www.contoso.com/\nmob:", "Addresses,CompanyNames,ContactNames,Departments,Emails,Faxes," +
+      "JobTitles,Locale,MobilePhones,Websites,WorkPhones")
+
+    val result2 = modelsTest(bytesAnalyzeBusinessCards, bytesDF3, true)
+    resultAssert(result2, "Dr. Avery Smith\nSenior Researcher\nCloud & Al Department\navery.smith@contoso.com\n" +
+      "https://www.contoso.com/\nmob:", "Addresses,CompanyNames,ContactNames,Departments,Emails,Faxes," +
+      "JobTitles,Locale,MobilePhones,Websites,WorkPhones")
+  }
+
+  lazy val analyzeInvoices: AnalyzeDocumentV3 = analyzeDocumentV3
+    .setPrebuiltModelId("prebuilt-invoice")
+    .setImageUrlCol("source")
+
+  lazy val bytesAnalyzeInvoices: AnalyzeDocumentV3 = analyzeDocumentV3
+    .setPrebuiltModelId("prebuilt-invoice")
+    .setImageBytesCol("imageBytes")
+
+  test("Prebuilt-invoice Basic Usage") {
+    val result1 = modelsTest(analyzeInvoices, imageDf4, false)
+    resultAssert(result1, "Contoso\nAddress:\nInvoice For: Microsoft\n1 Redmond way Suite\n1020 Enterprise Way\n",
+      "CustomerAddress,CustomerAddressRecipient,CustomerName,DueDate,InvoiceDate," +
+        "InvoiceId,Items,Locale,VendorAddress,VendorName")
+
+    val result2 = modelsTest(bytesAnalyzeInvoices, bytesDF4, true)
+    resultAssert(result2, "Contoso\nAddress:\nInvoice For: Microsoft\n1 Redmond way Suite\n1020 Enterprise Way\n",
+      "CustomerAddress,CustomerAddressRecipient,CustomerName,DueDate,InvoiceDate," +
+        "InvoiceId,Items,Locale,VendorAddress,VendorName")
+  }
+
+  lazy val analyzeReceipts: AnalyzeDocumentV3 = analyzeDocumentV3
+    .setPrebuiltModelId("prebuilt-receipt")
+    .setImageUrlCol("source")
+
+  lazy val bytesAnalyzeReceipts: AnalyzeDocumentV3 = analyzeDocumentV3
+    .setPrebuiltModelId("prebuilt-receipt")
+    .setImageBytesCol("imageBytes")
+
+  test("Prebuilt-receipt Basic Usage") {
+    val result1 = modelsTest(analyzeReceipts, imageDf2, false)
+    resultAssert(result1, "Contoso\nContoso\n123 Main Street\nRedmond, WA 98052\n123-456-7890\n" +
```

Comment:
nit: theres a bit of duplication in these tests (each assert happens twice), perhaps there's a nice way to reduce copy-paste here. Dont spend too much time worrying about it though :)